components/cache/adapters/array_cache_adapter.rst

```diff
@@ -24,5 +24,10 @@ method::
 
         // the maximum number of items that can be stored in the cache. When the limit
         // is reached, cache follows the LRU model (least recently used items are deleted)
-        $maxItems = 0
+        $maxItems = 0,
+
+        // implementation of Psr\Clock\ClockInterface (e.g. Symfony\Component\Clock\Clock)
+        // or null. If clock is provided, cache items lifetime will be calculated
+        // based on time provided by this clock
+        $clock = null
```

Comment:
I would shorten it a bit like this:

```
// optional implementation of the Psr\Clock\ClockInterface that will be used
// to calculate the lifetime of cache items (for example to get predictable
// lifetimes in tests)
$clock = null,

Reply:
Yes, this matches better the style of the existing docs, so I changed the original text by this while merging. Thanks.